components/trips/listTrips/ListTrips.js

```diff
@@ -42,13 +41,18 @@ export default class ListTrips extends Component {
         trips: firebaseFunctions.tripsCache,
         dataSource: ds.cloneWithRows(firebaseFunctions.tripsCache)
     }
+
   }
 
   componentDidMount() {
+      //this._childShowLoading.showLoading()
       firebaseFunctions.listenForTrips();
       let _this = this;
       _this.updateListDataSource(_this);
-      firebaseFunctions.addObserver('trips_changed', _this.updateListDataSource.bind(_this));
+      firebaseFunctions.addObserver('trips_changed', _this.updateListDataSource.bind(_this))
+
+      // Il est ou le callback??
+      //this._childShowLoading.hideLoading()
```

Comment:
cela marche quand je decalle vers le bas le component loading

          <ShowLoading
            ref={(child) => { this._childShowLoading = child; }}
          />